src/atsam/main.c

```diff
@@ -62,6 +66,10 @@ enable_pclock(uint32_t id)
 uint32_t
 get_pclock_frequency(uint32_t id)
 {
+#if CONFIG_MACH_SAME70
+    if (id == MCAN0_CLOCK_ID || id == MCAN1_CLOCK_ID)
+        return FREQ_CAN;
+#endif
```

Comment:
As before, it would be preferable to use a regular if instead of an ifdef.

Reply:
This isn't feasible here, unless we also define the clock ids for other chips, which would be a bit silly. Going to keep this one.